style.md

```diff
@@ -13,9 +13,12 @@ The redundancy reduction from removing the namespacing prefix is not useful enou
 ## Optional settings should have pointer Go types
 
 So we have a consistent way to identify unset values ([source][optional-pointer]).
+The exceptions are slices, maps, and booleans where the default should be false, in which case `omitempty` is sufficient and no pointer is needed ([source][no-pointer-for-slices] and [source][no-pointer-for-boolean]).
```

Comment:
On Tue, Jan 26, 2016 at 10:45:05AM -0800, Vish Kannan wrote:
> > +The exceptions are slices, maps, and booleans where the default
> >  should be false, in which case `omitempty` is sufficient and no
> >  pointer is needed ([source][no-pointer-for-slices] and
> >  [source][no-pointer-for-boolean]).
>
> I disagree with `bool` values. How can I express "don't change a
> value of a flag" if the flag was boolean? "false" does not always
> mean "don't change".

But sometimes false does mean “don't change” (and those are the
booleans where the default should be false).  For example, the source
I link is you asking for *bool → bool change [1] ;).

If my wording here comes across as “slices, maps, and booleans should
never use pointers” instead of the intended “slices, maps, and
booleans should not use pointers when Go's default for that type means
‘don't do anything’”, then I'm happy to push alternate wording.

[1]: https://github.com/opencontainers/specs/pull/290#discussion_r50296396